### PR TITLE
New revisions of chartdatum norway v2023b

### PR DESCRIPTION
**Pull request by himsve**
## New separation model model between ETRS89 ell. height and CD_NORWAY

The GeoTiff *no_kv_CD_above_Ell_ETRS89_v2023b.tif* is the lastest version of the vertical offset grid (separation model) from ETRS89 ell. heights (EPSG:4937) to CD_NORWAY depths (EPSG:9672).

* *no_kv_CD_above_Ell_ETRS89_v2023b* is a replacement of the current version *no_kv_CD_above_Ell_ETRS89_v2021a*
* *no_kv_CD_above_Ell_ETRS89_v2023b.tif* consists of two bands, vertical offsets (band 1) and vertical offset standard deviations (band 2). Band 2 is marked *vertical_offset_accuracy* in meters as GDAL Metadata. So far **vertical accuracy metadata is missing in PROJ**.
* The modified transformation is requested in EPSG, but not yet published.

### CODE REVIEW

File: copyright_and_licenses.csv [hunk 1]
```diff
@@ -195,7 +195,8 @@ nl_nsgi_rdcorr2018.tif,Nederlandse Samenwerking Geodetische Infrastructuur (NSGI
 nl_nsgi_rdtrans2018.tif,Nederlandse Samenwerking Geodetische Infrastructuur (NSGI),CC-BY-4.0,,
 nl_nsgi_README.txt,Disclaimed,Public domain,,
 no_kv_arcgp-2006-sk.tif,Kartverket (SK),CC-BY-4.0,1.11,
-no_kv_CD_above_Ell_ETRS89_v2021a.tif,Kartverket (SK),CC-BY-4.0,1.9,
+no_kv_CD_above_Ell_ETRS89_v2021a.tif,Kartverket (SK),CC-BY-4.0,,1.9
```
Comment (jjimenezshaw) [inline copyright_and_licenses.csv:198]: Here is modified, but the rest of the PR is deleting the file. That is not consistent.

**Review by jjimenezshaw — COMMENTED** [review]
Do you really want to delete the old file? Or add a new one, leaving the old one?
I see in EPSG the previous transformation https://epsg.org/transformation_9884/ETRS89-to-CD-Norway-depth-1.html
Are you going to delete it, or create a new one?
(In other countries it is usually a new one with a better accuracy, leaving the old one for consistency with previous data)

**Comment by jjimenezshaw** [comment]
> Are you going to delete it, or create a new one?

Actually I see it was already superseded by https://epsg.org/transformation_10130/ETRS89-to-CD-Norway-depth-2.html, but not deleted.

**Comment by rouault** [comment]
The metadata of the grid is incorrect. It should not be TYPE=VERTICAL_OFFSET_VERTICAL_TO_VERTICAL, but TYPE=VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL

I also think we should let the previous file for reproducability for older transformations. The EPSG supersession mechanism should ensure the new grid is used by default, once it has made its way through EPSG.

File: files.geojson [hunk 2]
```diff
@@ -2,36 +2,36 @@
...
-{ "type": "Feature", "properties": { "url": "https://cdn.proj.org/au_ga_README.txt", "name": "au_ga_README.txt", "source": "Geoscience Australia", "source_country": "Australia", "source_id": "au_ga", "source_url": "http://www.ga.gov.au", "file_size": 2213, "sha256sum": "82b411bb563187a5ac2c6ba6c2755a6277b6ab17841d56c91e60c6a163e0fc50" }, "geometry": null },
+{ "type": "Feature", "properties": { "url": "https://cdn.proj.org/ar_ign_README.txt", "name": "ar_ign_README.txt", "source": "Instituto GeogrÃ¡fico Nacional (IGN)", "source_country": "Argentina", "source_id": "ar_ign", "source_url": "https://www.ign.gob.ar", "file_size": 777, "sha256sum": "b3813ddb046480e541252ffcb5d4464b94bc013c321ccac8f06b8263a9a51d7f", "version_added": "1.15" }, "geometry": null },
```
Comment (jjimenezshaw) [inline files.geojson:5]: There are many changes due to different encodings. Apparently there is something different. Do you know what? It changes also the "sha"!

File: index.html [hunk 3]
```diff
@@ -497,8 +497,8 @@ <h2>Content</h2>
...
 <li><a style="word-break: break-word" href="br_ibge_CA61_003.tif">br_ibge_CA61_003.tif</a> - Brazil - Corrego Alegre 1961 - Corrego Alegre 1961 (EPSG:5524) to SIRGAS 2000 (EPSG:4674). Last modified: 2020-06-29</li>
 <li><a style="word-break: break-word" href="br_ibge_CA7072_003.tif">br_ibge_CA7072_003.tif</a> - Brazil - Corrego Alegre 1970-1972 - Corrego Alegre 1970-72 (EPSG:4225) to SIRGAS 2000 (EPSG:4674). Last modified: 2020-06-29</li>
-<li><a style="word-break: break-word" href="br_ibge_SAD69_003.tif">br_ibge_SAD69_003.tif</a> - Brazil - onshore south of 14°S and east of 53°W - SAD69 (EPSG:4618) to SIRGAS 2000 (EPSG:4674). Last modified: 2020-06-29</li>
-<li><a style="word-break: break-word" href="br_ibge_SAD96_003.tif">br_ibge_SAD96_003.tif</a> - Brazil - onshore south of 14°S and east of 53°W - SAD69(96) (EPSG:5527) to SIRGAS 2000 (EPSG:4674). Last modified: 2020-06-29</li>
+<li><a style="word-break: break-word" href="br_ibge_SAD69_003.tif">br_ibge_SAD69_003.tif</a> - Brazil - onshore south of 14�S and east of 53�W - SAD69 (EPSG:4618) to SIRGAS 2000 (EPSG:4674). Last modified: 2020-06-29</li>
```
Comment (jjimenezshaw) [inline index.html:500]: Another encoding change. But apparently in the other direction.

**Comment by rouault** [comment]
@himsve  You can drop the changes in the generated files (files.geojson, index.html) frmo the PR if that makes your life easier. I can easily refresh them on my side once this PR is merged. I suspect you might perhaps have some git issues with CRLF if you work on Windows.

File: no_kv/no_kv_README.txt [hunk 4]
```diff
@@ -60,7 +60,7 @@ Horizontal triangulation to transform coordinates from ETRS89 (EPSG:4258) to NGO
 *Source CRS*: EPSG:4937 (ETRS89 ellipsoidal heights)  
 *Target CRS*: EPSG:9672 (Chart datum Norway heights)
 
-* no_kv_CD_above_Ell_ETRS89_v2021a.tif
+* no_kv_CD_above_Ell_ETRS89_v2023b.tif
```
[the remark this answers is not in the document]
Comment (himsve) [reply no_kv/no_kv_README.txt:63]: It should be replaced

File: index.html (hunk 3 above)
Comment (himsve) [reply index.html]: I have done a new regeneration of index

File: files.geojson (hunk 2 above)
Comment (himsve) [reply files.geojson]: I have done a new regeneration of index

File: copyright_and_licenses.csv [hunk 5]
```diff
@@ -196,6 +196,7 @@ nl_nsgi_rdtrans2018.tif,Nederlandse Samenwerking Geodetische Infrastructuur (NSG
 nl_nsgi_README.txt,Disclaimed,Public domain,,
 no_kv_arcgp-2006-sk.tif,Kartverket (SK),CC-BY-4.0,1.11,
 no_kv_CD_above_Ell_ETRS89_v2021a.tif,Kartverket (SK),CC-BY-4.0,1.9,
+no_kv_CD_above_Ell_ETRS89_v2023b.tif,Kartverket (SK),CC-BY-4.0,1.16,
```
Comment (rouault) [inline copyright_and_licenses.csv:199]: this has missed the 1.16 window by a few days. Updating to:
```suggestion
no_kv_CD_above_Ell_ETRS89_v2023b.tif,Kartverket (SK),CC-BY-4.0,1.17,
```

**Comment by rouault** [comment]
@kbevers Is it fine to merge that right now in master, or will that cause trouble in your 1.16 release process?

**Pull request merged**